### PR TITLE
fix: revert removed type definitions

### PR DESCRIPTION
**Pull request by zardoy**
See https://github.com/iamkun/dayjs/issues/1360

### CODE REVIEW

**Comment by zardoy** [comment]
@iamkun Please could you merge this to revert the breaking change?
As you can remember I removed publicy available type definitions. This PR restores them.

File: types/plugin/duration.d.ts [hunk 1]
```diff
@@ -77,3 +86,5 @@ declare module 'dayjs' {
   export const duration: plugin.CreateDurationType;
   export function isDuration(d: any): d is plugin.Duration
 }
+
+type a = FnPa<plugin.AddDurationType>;
```
Comment (iamkun) [inline types/plugin/duration.d.ts:90]: what's this line used for, please?

**Comment by zardoy** [comment]
I wanted to add this line to `@deprecated` as a recommendation. I'm not sure how it was included into the last commit, but I have remove it. Check it out

File: types/plugin/duration.d.ts (hunk 1 above)
Comment (zardoy) [reply types/plugin/duration.d.ts]: I wanted to add this line to @deprecated as a recommendation. I'm not sure how it was included into the last commit, but I have remove it. Check it out

**Comment by zardoy** [comment]
Hey, @iamkun do you have any updates on that? :)

**Comment by iamkun** [comment]
THX

**Pull request merged**